src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountDecider.fs

```diff
@@ -0,0 +1,71 @@
+﻿module Deciders.BankAccountDecider
+
+open System
+open BankAccount
+
+type OpenBankAccount =
+    { BankAccountId: Guid
+      AccountNumber: string
+      ClientId: Guid
+      CurrencyISOCode: string }
+
+type RecordDeposit = { Amount: decimal; CashierId: Guid }
+type WithdrawCashFromATM = { Amount: decimal; AtmId: Guid }
+type CloseBankAccount = { Reason: string }
+
+type Command =
+    | OpenBankAccount of OpenBankAccount
+    | RecordDeposit of RecordDeposit
+    | WithdrawnCashFromATM of WithdrawCashFromATM
```

Comment:
Same typo: `Withdraw`, not `Withdrawn`?

Reply:
Fixed in https://github.com/oskardudycz/Ogooreck/pull/13/commits/e296c15b6cc63835f2dad69a8fd971470ae6a28f 🤦‍♂️

Reply:
ATM -> Atm

Reply:
ATM is an abbreviation; shouldn't we keep it in capital letters? 🤔

Reply:
No, not if >2 chars per FDG https://stackoverflow.com/a/30403453/11635
i.e. HTML would be ugly - thats used consistently throughtout the FW and .NET code in general
(the rules say DB and IO should not be Db and Io either, but I hate that and ignore it!)

Reply:
OK, updated that 🙂